x/tx/sonar-project.properties

```diff
@@ -1,7 +1,7 @@
-sonar.projectKey=cosmos-sdk-tx
+sonar.projectKey=cosmos-sdk-x/tx
```

Comment:
I don't think we should re-create a sonar project for that, so we should leave it as it was.

Reply:
reverted